### DISCUSSION TITLE
External frontend localization

### DISCUSSION THREAD

**Discussion by marins1897** [blockscout/blockscout#9962] [Q&A]
I builded my local Blockscout explorer with external Frontend.
To replicate my steps : 
- Bring up local blockchain network
- Cloned Blockscout/Frontend Github repo and made some styling changes to it
- Builded my custom Frontend image and started it locally
- Started Blockscout explorer using docker compose with -f external-frontend.yml up -d
- I also made configuration changes to ensure Blockscout is connected to external frontend etc.
- I can confirm that my external frontend is working as expected (I can see my network name, transactions, stats, charts etc.)

Now, my next goal is to translate this external frontend from EN to some other language, so I can have explorer fully available in my language (HR).
I saw post about changing default.pot and default.po in Blockscout/apps/block_scout_web/priv/gettext/EN/LC_MESSAGES but that post is 4 years old and I think it is only related to old frontend version of Blockscout and not to the external one.

What is the best and easiest way to include localization from EN to HR language in my external customized Blockscout Frontend?

**Comment by ekawibisono** [comment]
hello @marins1897 You mean change Language right ?

**Reply by marins1897** [reply]
Yes
